lib/riax_csv_example.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Riax.CSV do
+  alias NimbleCSV.RFC4180, as: CSV
+
+  @moduledoc """
+  Functions to read and distribute CSV among
+  Virtual Nodes for the example setup.
+  """
+  @doc """
+  Distribute the given path's CSV among Riak Nodes,
+  by storing it as row_number -> row values mapping.
+  """
+  def distribute_csv(path) do
+    :rpc.multicall(Riax.CSV, :store_csv, [path])
+  end
+
+  @doc """
+  Stores a CSV in the running node's assigned ring partitions.
+  Each of the CSV's rows are stored using the row's index number as key.
+  Eg. If I have "100" as key and the current ring status
```

Comment:
doesn't sound like the best choice for indexing, if you add an element in the middle all the indexes change?

Reply:
maybe it's not relevant for the example

Reply:
What would you suggest instead of indexing? 🤔

Reply:
For the example is not really relevant, but I could address it.

Reply:
I would use a field in the csv instead of the row number, but don't worry about it for now